### ISSUE TITLE
Implementation of Sale or Use/Salt List updated content (article & video) 

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2225]
**User story:** As a person who uses FEC data, I don't want to run afoul of the laws for using it and need to learn what I can use it for.

**Background** 
 This ticket is to finalize and publish the updated sale or use of information content (Record article and videos) drafted in #2115 and #2241 .

**Completion criteria**
- [x] Compile edits of OGC, RAD, Public Records and content team
- [x] Edit videos as needed and publish to FECTube (@jjonesfec)
1) - [x]  Salting video - Done
2) - [x] Use of contributor info video - Now in production

- [x] Publish new article, including videos embedded
- [x] Update the link at https://api.open.fec.gov/developers/ (needs a PR, simple to do per @lbeaufort)

**Include this in the larger redirect issues**
- [x] Add redirect for old brochure to the list (see #2350 )
- [x] Take old brochure off of transition (shtml & PDF)

**Comment by dorothyyeager** [comment]
Production on the article is done and just needs to have the videos embedded. Production on one of the of the videos has started, and the other video should be done by next week.

**Comment by dorothyyeager** [comment]
@jjonesfec has uploaded the first [video on YouTube](https://youtu.be/SSVGrpjQX4o) and it has been embedded in the article. Production on the second video will start soon.

**Comment by dorothyyeager** [comment]
@jjonesfec tells me production on the second video has begun and hopefully to be completed in next couple days. It will take less than an hour to update the article with that video and publish, so we're on course to finish this issue on time.

**Comment by dorothyyeager** [comment]
@jjonesfec reports that the production on the second video will take longer than expected but should be done early next week. With that news, will move this into the backlog for 7.5.

**Comment by dorothyyeager** [comment]
(Make that assigned tickets since he's working on it.)

**Comment by dorothyyeager** [comment]
The second video is just about done. Hoping to wrap this up today or tomorrow.

**Comment by dorothyyeager** [comment]
We have decided to publish the article with only one video embedded for now. The second video will undergo some editing and refinements, and our MVP principles dictate putting up what we have, which is perfectly adequate for updating a 1992 brochure badly in need of an update.

The article with embedded video on salting is at https://www.fec.gov/updates/sale-or-use-contributor-information/

Transition links to the old brochure have been updated. Adding the new page/old page to our redirect list.
